examples/porting/osal/lv_example_osal.c

```diff
@@ -71,7 +71,7 @@ static void increment_thread_entry(void * user_data)
 {
     LV_UNUSED(user_data);
     lv_obj_t * counter_label;
-    uint32_t press_count = 0;
+    int press_count = 0;
```

Comment:
Why was this converted to signed from unsigned?

Reply:
Changed to unsigned to match below "%u".